### DISCUSSION TITLE
Os compatability

### DISCUSSION THREAD

**Discussion by Googletrusting** [copy/v86#770] [Ideas]
Slax works.
Linux Lite 1.0.0 works. (must have ACPI enabled)
Linux from Scratch works.
Porteus works.
Knoppix works (must have ACPI enabled)
Windows 2.03 Works
GRML works. 
 Parsix works (Must have ACPI enabled.
MichalOS Works
NanoLinux Works
Element Os Works *Quite slow, takes ~5 minutes to boot, and needs at least 1 GB RAM and 64mb vram)
Chromixum works *Very slow takes ~8 minutes for the mouse to start moving and ~2 extra minutes for the wallpaper and shelf to load. Need at least 2 GB RAM and 256mb vram. ACPI must also be enabled. (Tested by @CorruptionVR )
AspireOS works 
Android Froyo works (tested by @CorruptionVR)
LinuX-gamers works

**Comment by Googletrusting** [comment]
Will update this when i test more oses.

**Reply by Ryan8085309853409** [reply]
Any os compatibility updates?

**Reply by Googletrusting** [reply]
Yes a minor update is that windows 2.03 works

**Reply by Googletrusting** [reply]
The reason why i cant really update that much is that it's getting harder to find an os that doesn't require a storage drive.

**Reply by spetterman66** [reply]
there are tons of livecd linux distros

**Reply by spetterman66** [reply]
(that other comment with same pfp but inverted is me on my phone)

**Reply by Googletrusting** [reply]
Alright so GRML and parsix works

**Reply by spetterman66** [reply]
try every os on https://archiveos.org

**Reply by CorruptionVR** [reply]
[Chromixum](https://archiveos.org/chromixium/) works
Took 7:50 to get mouse
9:29 wallpaper load
9:45 shelf loaded
done on windows 10 pc

settings
2048
256
ACPI enabled

**Reply by Alsefina** [reply]
Makes sense, most smaller linux distros work. Bodhi Linux was one of the first distros I tried on v86.

**Comment by Googletrusting** [comment]
Tried testing Element os But I get a kernel panic Error.

**Reply by spetterman66** [reply]
What does the kernel panic say?

**Reply by Googletrusting** [reply]
Kernel Panic! Not syncing: IO-APIC +timer doesnt work!

**Reply by Googletrusting** [reply]
Wait i fixed it. I needed to turn off ACPI becuase that caused the problem.

**Comment by CorruptionVR** [comment]
Android Froyo works
#962
